### PR TITLE
replace f string to format in exception handling for supporting all p…

### PR DESCRIPTION
**Pull request by Abhishek-121**
…ython versions

### CODE REVIEW

File: plugins/modules/inventory_intent.py [hunk 1]
```diff
@@ -616,7 +617,8 @@ def get_device_ids(self, device_ips):
 
             except Exception as e:
                 self.log("An error occurred while fetching the device from Cisco DNA Center")
-                raise Exception(f"'Error while fetching device from Cisco DNA Center' - {str(e)}")
+                error_message = "Error while fetching device from Cisco DNA Center - {0}".format(str(e))
```
Comment (madhansansel) [inline plugins/modules/inventory_intent.py:620]: can we use this error_message string in self.log as well?

File: plugins/modules/inventory_intent.py [hunk 2]
```diff
@@ -707,7 +709,8 @@ def get_diff_merged(self, config):
 
             except Exception as e:
                 self.log("An error occurred while Resyncing device in Cisco DNA Center")
-                raise Exception(f"'Error while Resyncing device in Cisco DNA Center' - {str(e)}")
+                error_message = "Error while Resyncing device in Cisco DNA Center - {0}".format(str(e))
```
Comment (madhansansel) [inline plugins/modules/inventory_intent.py:712]: can we use this error_message string in self.log as well?

Comment (Abhishek-121) [reply plugins/modules/inventory_intent.py:712]: yes, we can

File: plugins/modules/inventory_intent.py [hunk 3]
```diff
@@ -62,7 +62,7 @@
         description: Id path parameter. Device ID.Required for Deleting Device.
         type: str
       ipAddress:
-        description: Network Device's ipAddress.Required for Adding/Deleting Device except Meraki Devices.
+        description: Network Device's ipAddress.Required for Adding/Deleting/Resyncing Device except Meraki Devices.
```
Comment (madhansansel) [inline plugins/modules/inventory_intent.py:65]: Can you leave a space after "."

**Review by madhansansel — APPROVED** [review]

**Pull request merged**